src/core/Akka/IO/UdpConnection.cs

```diff
@@ -73,7 +73,7 @@ private void DoConnect(EndPoint address)
         {
             ReportConnectFailure(() =>
             {
-                _socket = new Socket(SocketType.Dgram, ProtocolType.Udp) { Blocking = false };
+                _socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp) { Blocking = false };
```

Comment:
Do we need to do IPV4 / IPV6 handling here?

Reply:
Maybe it should match the connect message address type?

Reply:
Matched the behavior to UdpListener and I caught a bug in the code